utils/abrt-action-analyze-java.c

```diff
@@ -162,12 +162,11 @@ work_out_list_of_remote_urls(struct sr_java_stacktrace *stacktrace)
         thread = thread->next;
     }
 
-    if (remote_files_csv->buf[0] != '\0')
+    if (remote_files_csv->str[0] != '\0')
     {
-        return libreport_strbuf_free_nobuf(remote_files_csv);
+        return g_string_free(remote_files_csv, FALSE);
```

Comment:
This would result in a double free, as far as I can see. Try `g_steal_pointer()`.